### ISSUE TITLE
Track Tuple of Values at Once (Multi Dimensional Datapoints)

### ISSUE THREAD

**Issue by fmagin** [SamAmco/track-and-graph#148]
First off, I really like this app, because its flexibility allows me to track various different things all in one interface. This basically replaces a ton of questionable other kind of closed source "tracking" or "journal" apps.

One limitation I ran into a few times already is that I want to track something that inherently has multiple dimensions. The current workaround I use is to abuse decimal places, i.e. to save the integer tuple (x,y) I save the numeral "x.y", so e.g. for (10, 5) I save the numeral `10.5`. This is obviously a horrible hack.

The current thing I want to track is fitness/weight training progress, and the most elegant idea IMO would be to have one tracker for each kind of exercise that tracks the repetitions and the weight, e.g. `(10 reps, 20kg)`. 

This raises various implementation questions, and I by no means assume that this will be a trivial change to implement, so for now it would be interesting how many others have run into this limitation and would be interested.

**Comment by tomthe** [comment]
Yes, I would also like to use several values at once. I used https://github.com/tomthe/quantifymyself until now, where I can log 4 values with one button. I can work around this by using 4 trackers instead of one, but this has also several drawbacks (loose the relation between those 4 values; very tedious if I want to track something in the past: I have to change the time 4 times...)

**Comment by SamAmco** [comment]
Hi please see my last response on #144. TLDR I am aware of this problem and am trying to implement changes that will fix it but perhaps not in the way you expect.
